dotcom-rendering/scripts/webpack/frontend.js

```diff
@@ -48,7 +48,9 @@ const commonConfigs = ({ platform }) => ({
 		}),
 		// Does not try to require the 'canvas' package,
 		// an optional dependency of jsdom that we aren't using.
-		new webpack.IgnorePlugin({ resourceRegExp: /^canvas$/ }),
+		new webpack.IgnorePlugin({
+			resourceRegExp: /^(canvas|bufferutil|utf-8-validate)$/,
```

Comment:
'bufferutil' and 'utf-8-validate' are both optional dependencies of 'ws' which are not installed by jsdom. As these are not used, we can filter out their requires, reducing noise on the server build.

Reply:
Could we update the code comment to reflect this?